### PR TITLE
chore: Adapt to qt6

### PR DESCRIPTION
**Pull request by 18202781743**
Adapt to cmake.

Issue: https://github.com/linuxdeepin/dtk/issues/86

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,22 @@ cmake_minimum_required(VERSION 3.7)
 
 set(VERSION 4.0)
 
+set(DTK_VERSION "5" CACHE STRING "Dtk Version")
+
+if (DTK_VERSION STREQUAL "5")
+    set(QT_VERSION_MAJOR "5")
+    set(DTK_VERSION_MAJOR "")
```
Comment (kegechen) [inline CMakeLists.txt:13]: if (DTK_VERSION STREQUAL "5")
set(DTK_VERSION_MAJOR "")
😕

Comment (18202781743) [reply CMakeLists.txt:13]: DTK_VERSION打包传进来的，
而DTK_VERSION_MAJOR主要是我们不支持加5，，

File: CMakeLists.txt [hunk 2]
```diff
@@ -6,6 +6,22 @@ cmake_minimum_required(VERSION 3.7)
 
 set(VERSION 4.0)
```
Comment (FeiWang1119) [inline CMakeLists.txt:7]: ?

File: CMakeLists.txt [hunk 3]
```diff
@@ -6,6 +6,22 @@ cmake_minimum_required(VERSION 3.7)
...
 
+set(DTK_VERSION "5" CACHE STRING "Dtk Version")
```
Comment (kegechen) [inline CMakeLists.txt:9]:
```suggestion
set(DTK_VERSION_MAJOR "5" CACHE STRING "Dtk Version")
```

**Review by kegechen — APPROVED** [review]

**Review by FeiWang1119 — APPROVED** [review]

**Pull request merged**